app/models/user.rb

```diff
@@ -1,6 +1,7 @@
-class User < ActiveRecord::Base
-  acts_as_token_authenticatable
+class User < ApplicationRecord
+  has_secure_password
 
-  devise :database_authenticatable, :registerable,
-    :recoverable, :trackable, :validatable
+  validates :email, presence: true
+  validates :password, length: { minimum: 6 }
```

Comment:
To update a user's attribute he always needs to pass a password. Was it intended? Looks like if you need to update a full_name for example, you don't need to provide a password.

```ruby
validates :password, length: { minimum: 6 }, if: -> { new_record? || password_digest_changed? }
```